### ISSUE TITLE
"Error building bundles: rpm not found" when using customized repo names

### ISSUE THREAD

**Issue by yewwayne** [clearlinux/mixer-tools#659]
Mixer build with "clear" repo replaced with something else runs into the following error:
```
Installing all bundles to full chroot

TIMINGS
  BUILD BUNDLES 0s
TOTAL: 0s
2019/11/29 09:41:04 ERROR: Error building bundles: rpm not found: [snip]/update/image/10/full/var/cache/yum/clear/clear-317a091ba1f3af6d/packages/filesystem-3.0.14-191.x86_64.rpm
```
When inspecting the filesystem, the RPM exists but under a different cache folder that corresponds to the custom repo name, not "clear-*" as assumed by the following line:
https://github.com/clearlinux/mixer-tools/blob/8e8fda08c03d03069b2f4b7e99417c1a69ee5c7d/builder/bundles.go#L547

Related issue: #654

**Comment by yewwayne** [comment]
A straightforward fix here would be to add `--destdir` for DNF to download directly to a specific folder instead of the internal cache.

**Comment by reaganlo** [comment]
Thanks for the suggestion. Will look into it.

Is your repo local or over the network?

**Comment by yewwayne** [comment]
Points to a Koji server, so network. FYI I've worked around the issue by keeping the "clear" repo name and just change its url.

**Comment by reaganlo** [comment]
Cool. That's what I was going to suggest as the workaround. But looks like you are on it. +1

**Comment by reaganlo** [comment]
Fixed in [v6.1.2](https://github.com/clearlinux/mixer-tools/tree/v6.1.2)
